ui/ducks/metamask/metamask.js

```diff
@@ -209,7 +209,7 @@ const toHexWei = (value, expectHexWei) => {
 };
 
 // Action Creators
-export function updateTransactionGasFees({
+export function updateTransaction({
```

Comment:
[Q] Why is this function being renamed? I am not a fan of this change because it mimics the naming of a function that previously updated the transaction in state in the back-end... which made me think all of its usages in this were that old function.  And it seems like this is still only updating gas related fields? So why was the name changed?

Reply:
I renamed it to avoid name collision (in this file) with updateTransactionGasFees of txController.

Reply:
I changed the name to `updateGasFees` [here](https://github.com/MetaMask/metamask-extension/pull/13646/commits/70f994ad10acfb26327d4dd1398c0195e91ab258)